### PR TITLE
adding Feb/17 mm

### PR DESCRIPTION
**Pull request by oferShinaar**

### CODE REVIEW

File: meetings/overlay_mm_17_feb_2021.adoc [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+** flag to activate the creation of CRC checksum on linker side
+** let us state the chosen CRC algo (just pick one)
+** We need better names for MIN/MAX_GROUP_SIZE, and we need to align it with the
+draft.
+
+* Check if we had crc32 hook on the hook section. If missing, we should add it.
+** *[OS]*: The CRC is not part of the *HLD* same as all hooks. They are part of
+the implementation (*LLD*). On the _riscv-overlay-software-standard-draft.adoc_
+we only specified the *must* have hooks. CRC hook is an optional per-application
+design.
```
Comment (craigblackmore) [inline meetings/overlay_mm_17_feb_2021.adoc:34]: Might be worth adding a short an explanation of why the crc32 is there, what it is the crc32 of and who uses it. Either in the requirements document or in HLD 2.2.1.

File: meetings/overlay_mm_17_feb_2021.adoc [hunk 2]
```diff
@@ -0,0 +1,63 @@
+== General summary
+* We will keep the meeting on the same time slot:
+*_Wednesday 6:00 am PST/2 pm UK/4 pm ISR/10 pm China_*
+* After HLD review will be complete we will start working on publish patches
+to llvm and GNU.
+
+== Action items
+* 2.7.1.5 add bullets
+* 2.7.2 elaborate that *ovliput* are sections arrived from *overlay attribute*?
```
Comment (craigblackmore) [inline meetings/overlay_mm_17_feb_2021.adoc:9]: `ovlinput`

File: meetings/overlay_mm_17_feb_2021.adoc [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+* We will keep the meeting on the same time slot:
+*_Wednesday 6:00 am PST/2 pm UK/4 pm ISR/10 pm China_*
+* After HLD review will be complete we will start working on publish patches
+to llvm and GNU.
+
+== Action items
+* 2.7.1.5 add bullets
+* 2.7.2 elaborate that *ovliput* are sections arrived from *overlay attribute*?
+* 2.7.2.1 /2 lets mentioned that grouping tool is optional and number '3' is
+the default
```
Comment (craigblackmore) [inline meetings/overlay_mm_17_feb_2021.adoc:11]: `lets mentioned that grouping tool is optional and number '3' is
the default`
change to:
`mention that linker autogrouping is the default and that grouping file and tool are optional. Using both a grouping file and grouping tool is an error.`

**Review by craigblackmore — APPROVED** [review]
Other than minor comments, LGTM.

File: meetings/overlay_mm_17_feb_2021.adoc (hunk 1 above)
Comment (oferShinaar) [reply meetings/overlay_mm_17_feb_2021.adoc:34]: it is only mentioned in the hld toolchain section, right ?
If so I think we should remove it

File: meetings/overlay_mm_17_feb_2021.adoc (hunk 2 above)
Comment (oferShinaar) [reply meetings/overlay_mm_17_feb_2021.adoc]: done in #ade4

File: meetings/overlay_mm_17_feb_2021.adoc (hunk 3 above)
Comment (oferShinaar) [reply meetings/overlay_mm_17_feb_2021.adoc]: done in #ade4

File: meetings/overlay_mm_17_feb_2021.adoc (hunk 1 above)
Comment (craigblackmore) [reply meetings/overlay_mm_17_feb_2021.adoc:35]: The first reference to crc32 is in Figure 2 in the RT-Engine HLD.

Comment (oferShinaar) [reply meetings/overlay_mm_17_feb_2021.adoc:35]: You are right. It is in the picture, that's why I did not find it.
I will update the picture.

**Pull request merged**